go.mod

```diff
@@ -7,36 +7,32 @@ require (
 	github.com/Microsoft/go-winio v0.4.14 // indirect
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
-	github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4 // indirect
 	github.com/aws/aws-sdk-go v1.19.6
+	github.com/aws/aws-sdk-go-v2 v0.23.0 // indirect
```

Comment:
This and `go.sum` at the root level probably shouldn't change, since all the changes were in the `v2/` directory which has its own `go.mod` and `go.sum`. Can you revert this?